src/dune_rules/coq/coq_rules.ml

```diff
@@ -11,7 +11,7 @@ open Memo.O
 (* Utilities independent from Coq, they should be eventually moved
    elsewhere; don't mix with the rest of the code. *)
 module Util : sig
-  val include_flags : Lib.t list -> _ Command.Args.t
+  val include_flags : (Lib.t * bool) list -> _ Command.Args.t
```

Comment:
As far as I know, Coq does not have a notion of hidden includes, so this function should probably stay as before.